website/forms/php/filters/validation/force-mimetype-from-fs.md

```diff
@@ -0,0 +1,26 @@
+---
+id: force-mimetype-from-fs
+title: Force mimetype from FS
+---
+
+Force mimetype validation from filesystem values.
+
+By default, mime-types are validated from the filesystem mimetype.
+
+However, in case the file is not present on the filesystem for any reason, this will fall back to the POST-provided mimetype.
+
+Using this filter, you can force Eightshift Forms to fail every file upload where it can't validate the mimetype from the filesystem.
```

Comment:
```suggestion
Forces MIME type validation based on the filesystem values. This is the default behavior. In case the file does not exist, the behavior falls back to validating using the MIME type from the POST object.

If `true` is returned from this filter, the Forms file upload will fail if the MIME type cannot be matched from the filesystem.
```